drivers/espi/host_subs_npcx.c

```diff
@@ -502,51 +502,51 @@ static void host_port80_init(void)
 static void host_cus_opcode_enable_interrupts(void)
 {
 	/* Enable host KBC sub-device interrupt */
-#if defined(CONFIG_ESPI_PERIPHERAL_8042_KBC)
-	irq_enable(DT_INST_IRQ_BY_NAME(0, kbc_ibf, irq));
-	irq_enable(DT_INST_IRQ_BY_NAME(0, kbc_obe, irq));
-#endif
+	if (IS_ENABLED(CONFIG_ESPI_PERIPHERAL_8042_KBC)) {
+		irq_enable(DT_INST_IRQ_BY_NAME(0, kbc_ibf, irq));
+		irq_enable(DT_INST_IRQ_BY_NAME(0, kbc_obe, irq));
+	}
 
 	/* Enable host PM channel (Host IO) sub-device interrupt */
-#if defined(CONFIG_ESPI_PERIPHERAL_HOST_IO) || \
-				defined(CONFIG_ESPI_PERIPHERAL_EC_HOST_CMD)
-	irq_enable(DT_INST_IRQ_BY_NAME(0, pmch_ibf, irq));
-#endif
+	if (IS_ENABLED(CONFIG_ESPI_PERIPHERAL_HOST_IO) ||
+		IS_ENABLED(CONFIG_ESPI_PERIPHERAL_EC_HOST_CMD)) {
```

Comment:
linux kernel style is to align with the parenthesis on continutations:

> Descendants are always substantially shorter than the parent and are placed substantially to the right. A very commonly used style is to align descendants to a function open parenthesis.

https://www.kernel.org/doc/html/v5.8/process/coding-style.html#breaking-long-lines-and-strings

Reply:
Done.